### ISSUE TITLE
is there any implemtion with avx2 my cpu doesnt support it :(

### ISSUE THREAD

**Issue by aliersahan** [crashoz/uuid_v4#19]

**Comment by duckdoom5** [comment]
As the description of the project mentions , you can use a library like simd-everywhere to support other architecture's without the need to modify any code

**Comment by TrueWodzu** [comment]
@duckdoom5 Could you be a bit more elaborate how to use simd-everywhere? Do I need to include simd-everywhere header in your library?

**Comment by duckdoom5** [comment]
> @duckdoom5 Could you be a bit more elaborate how to use simd-everywhere? Do I need to include simd-everywhere header in your library?

Yeah, so what you do is include the simde library in your project, following the guide on their GitHub page. (As mentioned there, make sure to select the right architecture).

Then you can use the options they have to replace the calls to simd instructions in this uuid library. The easiest is probably to just put `simde_` in front. So it would become `simde_my_simd_instruction();`

_FYI, this isn't my library. I just answered the question_

**Comment by TrueWodzu** [comment]
@duckdoom5 Thanks
